src/App.php

```diff
@@ -300,8 +301,8 @@ public function caughtException(\Throwable $exception): void
         // remove header
         $this->layout->template->tryDel('Header');
 
-        if (($this->isJsUrlRequest() || strtolower($_SERVER['HTTP_X_REQUESTED_WITH'] ?? '') === 'xmlhttprequest')
-                && !isset($_GET['__atk_tab'])) {
+        if (($this->isJsUrlRequest() || $this->request->getHeaderLine('HTTP_X_REQUESTED_WITH') === 'xmlhttprequest')
```

Comment:
![image](https://github.com/atk4/ui/assets/2228672/5f3fe01e-f422-4887-a3f7-c4337ecdcb34)

the removed `strtolower` caused 500 HTTP code on AJAX failure to be returned instead of 200

I will fix this, headers are case sensitive, so no `strtolower` is wanted but instead compare to `XMLHttpRequest` value